bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Flight/Maneuver.cpp

```diff
@@ -161,8 +161,22 @@ inline void OpenSpaceToolkitAstrodynamicsPy_Flight_Maneuver(pybind11::module& aM
             )doc"
         )
         .def_static(
-            "constant_mass_flow_rate_profile",
-            &Maneuver::ConstantMassFlowRateProfile,
+            "from_tabulated_dynamics",
```

Comment:
Yeah you're right, I had changed it back because I wanted to be able to 
`using TabulatedDynamics = ostk::astrodynamics::dynamics::Tabulated;`
inside the `Maneuver.hpp/cpp` file and it conflicts with declaring the `Maneuver TabualtedDynamics(XX);` method, but I'm changing it back to maintain a consistent API across OSTk.